### PR TITLE
Moved Project and Organization

### LINKED ISSUE #527: Move `Project` class into `jvmMain`, introduce `ProjectDto`

### PR DESCRIPTION
**Pull request by nulls**
It closes #527

### CODE REVIEW

File: save-backend/src/test/kotlin/com/saveourtool/save/backend/controller/AgentsControllerTest.kt
Comment (akuleshov7) [inline save-backend/src/test/kotlin/com/saveourtool/save/backend/controller/AgentsControllerTest.kt:71]: Time is not needed on AgentStatus? Why?

File: save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/Contest.kt
Comment (akuleshov7) [inline save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/Contest.kt:57]: I think the previous version, when we just had a typealias with expect/actual implementation looked better...

**Review by akuleshov7 — APPROVED** [review]
Looks good, we will definitely get errors because of this change, but the logic get's better, I love it

**Comment by akuleshov7** [comment]
Will need to stop other's development, until you merge it

File: save-backend/src/test/kotlin/com/saveourtool/save/backend/controller/AgentsControllerTest.kt
Comment (nulls) [reply save-backend/src/test/kotlin/com/saveourtool/save/backend/controller/AgentsControllerTest.kt:71]: it's set as LocalDateTime.now() in constructor. We set only in two places another values and it's a test

File: save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/Contest.kt
Comment (nulls) [reply save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/Contest.kt:57]: yeap, it will be great to have alias in kotlinx.serialization instead of a dedicated class

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationAdminView.kt
Comment (nulls) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationAdminView.kt:169]: fixed

**Comment by nulls** [comment]
> Will need to stop other's development, until you merge it

will merge after #1454 -- will wait to avoid merge conflicts

**Pull request merged**
